iOS/issue-tracker/issue-tracker/Network/LoginHelper.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import Combine
+
+class LoginHelper {
+    
+    private let callbackURLScheme = "issueTracker"
+    private let gitHubLoginUrlString = "https://github.com/login/oauth/authorize?client_id=04fb3475fc652d5304a3"
+    
+    func getCallbackURLscheme() -> String {
```

Comment:
```suggestion
    func callbackURLscheme() -> String {
```

Swift 에서는 값을 반환할때 get을 사용하지 않고 단순히 명사를 사용합니다. 
set은 Swift 도 네이밍 그대로 사용합니다. 참고 부탁드려요.

Reply:
다시 한 번 참고하겠습니다!